server/plugin/plugin.go

```diff
@@ -1095,3 +1165,21 @@ func (p *Plugin) getUsername(mmUserID string) (string, error) {
 
 	return "@" + info.GitHubUsername, nil
 }
+
+func (p *Plugin) useGitHubClient(info *GitHubUserInfo, toRun func(info *GitHubUserInfo, token *oauth2.Token) error) error {
+	err := toRun(info, info.Token)
+	if err != nil {
+		fmt.Println(err.Error())
```

Comment:
Why are we printing an error to stdout?

Reply:
@wiggin77 Replaced the print statement with a log. Please re-review